### PR TITLE
fix: wrong log env vars

### PR DESCRIPTION
**Pull request by xavierchow**
resolve: #79

### CODE REVIEW

File: server/config.local.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ const env = require('env-var-defaults').injected(['test', 'development', 'dev',
  */
 module.exports = {
   // Server.
-  port: env.get('HELLO_PORT').asInt(),
   rootCaFile: env.get('ROOT_CA_FILE').asString(),
 
   // Log.
+  logName: env.get('LOG_NAME').asString(),
```
Comment (Zacharysp) [inline server/config.local.js]: should we add default value for logName also the syslogHost and etc?

Comment (xavierchow) [reply server/config.local.js]: let DevOps decide them, failing straightforward would be better if any of them missing.

Comment (Zacharysp) [reply server/config.local.js]: cool

File: server/boot/lib-model.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ module.exports = app => {
   app.logger = app.lib.logger = loggerFactory.init({
     logName: 'wcl-service',
```
Comment (Zacharysp) [inline server/boot/lib-model.js]: should app.get

Comment (xavierchow) [reply server/boot/lib-model.js]: thx, updating

**Pull request merged**